src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -545,17 +627,20 @@ $spacing: 4px;
 
 	&--button-variant-h-grouped {
 		&:first-of-type {
-			border-top-left-radius: var(--border-radius-large);
-			border-bottom-left-radius: var(--border-radius-large);
+			border-top-left-radius: var(--border-radius-nc-radio-switch);
```

Comment:
These values need to be `2px` larger than the ones for the inner buttons to account for the border-thickness.

Reply:
Just for reference, here you see this gap:
![image](https://user-images.githubusercontent.com/2496460/229503691-505dfdca-397e-4808-9f01-95bee9d81331.png)

Reply:
:+1: